cypress/e2e/po/edit/provisioning.cattle.io.cluster/create/cluster-create.po.ts

```diff
@@ -1,5 +1,6 @@
 import ToggleSwitchPo from '@/cypress/e2e/po/components/toggle-switch.po';
 import ClusterManagerCreateImportPagePo from '@/cypress/e2e/po/edit/provisioning.cattle.io.cluster/cluster-create-import.po';
+import BannersPo from '~/cypress/e2e/po/components/banners.po';
```

Comment:
Can we use the @ import, rather than ~ ?

Reply:
Fixed!